po/es.po

```diff
@@ -30,29 +30,17 @@ msgstr ""
 "Plural-Forms: nplurals=2; plural=(n != 1)\n"
 "X-Generator: Gtranslator 3.34.0\n"
 
-msgid " To stop these messages, say STOP."
-msgstr ""
+msgid "To stop these messages, say STOP."
+msgstr "Si no quieres recibir esta mensajes, diga STOP"
 
 msgid "You are receiving this email because you subscribed to notifications from Genie. To stop, open the following link: %s"
-msgstr ""
+msgstr "Recibiste este correo electrónico porque te suscribiste a las notificaciones de el Covid Genio. Para detener, abra el siguiente enlace: %s"
 
 msgid "You are receiving this email because you subscribed to notifications from Genie. To stop, you can <a href=\"%s\">unsubscribe</a>."
-msgstr ""
-
-msgid "Sorry, this is not a valid zip code, please re-enter it."
-msgstr ""
-
-msgid "Sorry, you do not need a second dose for the johnson & johnson vaccine."
-msgstr ""
-
-msgid "Is this your first or second dose?"
-msgstr ""
-
-msgid "What is your zip code?"
-msgstr "¿Cuál es su código postal?"
+msgstr "Recibiste este correo electrónico porque te suscribiste a las notificaciones de el Covid Genio. Para detener, puedes <a href=\"%s\">unsubscribe</a>."
```

Comment:
You accidentally deleted some strings to translate (maybe started off a slightly stale branch?)